source/api_documentation/call_api/_get_call_transcript_caller.rst

```diff
@@ -0,0 +1,23 @@
+.. container:: endpoint-long-description
+
+  .. rubric:: Examples
+
+  Get transcript for the given call record `CUID-33` with just the caller included
+
+  Endpoint:
+
+  ``https://mynetwork.invoca.net/call/transcript/CUID-33?transcript_format=caller``
+
+  Format: application/json
+
+  Response Code: 200
+
+  Response Body:
+
+  .. code-block:: json
+
+     [
+       {
+        caller: "Orange, Awesome thank you, Have a good one"
```

Comment:
same here: should be 2 spaces for indent

Reply:
Ah good catch! The spacing was off in all the examples so I have updated them to be 2 spaces for an indent!